app/views/layouts/application.html.erb

```diff
@@ -16,5 +16,10 @@
     <%= render 'shared/flashes' %>
 
     <%= yield %>
+
+     <p class="notice"><%= notice %></p>
```

Comment:
I think notice & alert are already being rendered in by `<%= render 'shared/flashes' %>`